File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.cpp
class Solution {
public:
    bool checkIfPangram(string sentence) {
        int n = sentence.size();
        if(n<26) return false;
        sort(sentence.begin(),sentence.end());
        string ans = "abcdefghijklmnopqrstuvwxyz";
        int i=0;
        int j = 0;
        while(i<n && j<26){
            if(sentence[i]<ans[j]){
                i++;
            }
            else if(sentence[i]==ans[j]){
                i++;
                j++;
            }
            else i++;
        }
        return j==26;
    }
};

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        int n = sentence.length();
        if(n<26) return false;
        int[] arr = new int[26];
        for(int i=0;i<n;i++){
            char c = sentence.charAt(i);
            int x = c - 'a';
            arr[x]++;
        }
        for(int i=0;i<26;i++){
            if(arr[i]==0) return false;
        }
        return true;
    }
}